pkg/secreturl/client.go

```diff
@@ -0,0 +1,20 @@
+package secreturl
+
+// Client is a simple interface for Vault-like things such as `vault.Client` or a file system we can use to
+// access secret files and values in helm.
+//go:generate pegomock generate github.com/jenkins-x/jx/pkg/vault Client -o mocks/vault_client.go
```

Comment:
name package name in the mock generator needs to be updated

github.com/jenkins-x/jx/pkg/vault => github.com/jenkins-x/jx/pkg/secreturl Client -o mocks/secreturl_client.go